### PR TITLE
[Docs] Enhance docs for /api/user/token

### PR DESCRIPTION
**Pull request by coder12git**
**Notes for Reviewers**

This PR fixes ##6436

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit db1201ea9b09c958e36e48a985a49b99d00b71e4 at: https://65171852b441050cc31b3426--meshery-docs-preview.netlify.app

File: docs/_data/swagger.yml [hunk 1]
```diff
@@ -15628,20 +15628,35 @@ paths:
...
+                                example: Meshery
+                            token:
+                                type: string
+                                example: 
+                                
             description: |-
-                Returns token from the actual provider in a file
-                response:
-                200:
+                Returns token from the actual provider in a file.
```
Comment (leecalcote) [inline docs/_data/swagger.yml:15656]:
```suggestion
                Returns API token from the Provider in a file.
```

File: docs/_data/swagger.yml [hunk 2]
```diff
@@ -15628,20 +15628,35 @@ paths:
                 - SystemAPI
     /api/user/token:
         post:
+            responses: 
+               "200":
+                 description: Returns a token
```
Comment (leecalcote) [inline docs/_data/swagger.yml:15633]:
```suggestion
                 description: Returns an API token from the current Provider
```

**Review by leecalcote — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit f8466c7868495672eca704ffef44a4683fb7a425 at: https://6517740a3749894b2ae2961c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1a6785fcbf47eefed86e24467e179754959b3ff6 at: https://651798a42e43b25e86393a2a--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**